pkg/service/replay/replay.go

```diff
@@ -763,16 +769,22 @@ func (r *Replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 
 	if testSetStatus == models.TestSetStatusFailed || testSetStatus == models.TestSetStatusPassed {
 		if testSetStatus == models.TestSetStatusFailed {
-			pp.SetColorScheme(models.GetFailingColorScheme())
+			FailingColorScheme := models.GetFailingColorScheme()
+			//Keeps all values in test report blue
+			FailingColorScheme.Float = pp.Blue | pp.Bold
+			pp.SetColorScheme(FailingColorScheme)
 		} else {
-			pp.SetColorScheme(models.GetPassingColorScheme())
+			PassingColorScheme := models.GetPassingColorScheme()
+			//Keeps all values in test report blue
+			PassingColorScheme.Float = pp.Blue | pp.Bold
+			pp.SetColorScheme(PassingColorScheme)
 		}
 		if testReport.Ignored > 0 {
-			if _, err := pp.Printf("\n <=========================================> \n  TESTRUN SUMMARY. For test-set: %s\n"+"\tTotal tests: %s\n"+"\tTotal test passed: %s\n"+"\tTotal test failed: %s\n"+"\tTotal test ignored: %s\n <=========================================> \n\n", testReport.TestSet, testReport.Total, testReport.Success, testReport.Failure, testReport.Ignored); err != nil {
+			if _, err := pp.Printf("\n <=========================================> \n  TESTRUN SUMMARY. For test-set: %s\n"+"\tTotal tests: %s\n"+"\tTotal test passed: %s\n"+"\tTotal test failed: %s\n"+"\tTotal test ignored: %s\n"+"\tTime Taken: %s\n <=========================================> \n\n", testReport.TestSet, testReport.Total, testReport.Success, testReport.Failure, testReport.Ignored, timeTaken.Seconds()); err != nil {
```

Comment:
It will be good if we display the duration according to value. If it is less than a second lets show it in milliSeconds, if it is less than a minute lets show in seconds, if it is less than an hours lets show it in minute, If it is more than an hour lets show it in hrs. Lets have unit(ms/s/min/hr) while printing too

Reply:
I have changed the code according to the above convention, and have changed the code to show ms/s/min/hr. I'm storing this value in a string with the unit and printing it.
Currently it looks like this:
![image](https://github.com/user-attachments/assets/4ed396ec-4488-43fc-af38-bf117f8b9769)
If we want to remove the quotes around time taken, we need to change this portion of the [code](https://github.com/keploy/keploy/blob/c306030272404fc19b2a1a8783ea1ceefd123154/pkg/service/replay/replay.go#L899). It needs to written with a if else block. I have gone through go pp package that we're using for printing and it doesn't seem to have any options to prevent the quotes for strings. Should I go ahead or stick with this format?

Reply:
yep we continue quotes only not a problem

Reply:
or what we can do is add the unit in key like Time Taken(ms): 102

Reply:
Lets go with string only because when you show complete test summary it will be difficult if we add unit in the key.

Reply:
In the complete test run, it would be difficult to integrate this format as time for each test set can vary from ms to hr. It would lead to inconsistent UI between the individual test report format and the complete test report format.
![image](https://github.com/user-attachments/assets/fd53bbb4-1ab1-448b-8747-97d53ea571c9)

Reply:
Got it